types/boom/index.d.ts

```diff
@@ -32,14 +34,17 @@ declare namespace Boom {
         /** headers - an object containing any HTTP headers where each key is a header name and value is the header content. (Limited value type to string https://github.com/hapijs/boom/issues/151 ) */
         headers: {[index: string]: string};
         /** payload - the formatted object used as the response payload (stringified). Can be directly manipulated but any changes will be lost if reformat() is called. Any content allowed and by default includes the following content: */
-        payload: {
-            /** statusCode - the HTTP status code, derived from error.output.statusCode. */
-            statusCode: number;
-            /** error - the HTTP status message (e.g. 'Bad Request', 'Internal Server Error') derived from statusCode. */
-            error: string;
-            /** message - the error message derived from error.message. */
-            message: string;
-        }
+        payload: IPayload;
+    }
+    export interface IPayload {
```

Comment:
Great, thanks @andy-ms.

Also as there's no `tslint.json` in boom it'll just be passing on the `tsc` if I've understood the readme correctly?  And tried it now on 6.10.2, the latest of the node LTS so that shouldn't be the problem.  Have opened an issue on dtslint: https://github.com/Microsoft/dtslint/issues/29